ngraph/frontend/onnx_import/src/op/constant.cpp

```diff
@@ -165,6 +167,171 @@ namespace ngraph
                     default: throw error::tensor::invalid_data_type{tensor};
                     }
                 }
+
+                template <typename T>
+                std::vector<T> get_dense_vector(const std::vector<T>& values,
+                                                const std::vector<int64_t>& indices,
+                                                const size_t size)
+                {
+                    std::vector<T> dense_values(size);
+                    for (size_t i = 0; i < values.size(); ++i)
```

Comment:
This loop requires that `values.size() == indices.size()`, otherwise you might get some weird behavior here. Are those vetctors always the same size?

Also, it's possible that `indices.at(i)` returns something larger than `dense_values.size()` and you will get an exception.

Reply:
Done